core/src/main/resources/reference.conf

```diff
@@ -59,11 +59,34 @@ akka {
 
     # Note that without further configuration, all other actors run on the default dispatcher
   }
+
+  coordinated-shutdown.phases {
+    # This phase is used to give time to Cromwell to abort all workflows upon shutdown. It's only used if system.abort-jobs-on-terminate = true
+    abort-all-workflows {
+      # /!\ This depends-on field should NOT be modified
+      depends-on = [service-unbind]
```

Comment:
What about something like [this?](https://groups.google.com/forum/#!topic/akka-user/A4MB5sNLm-s)

This person had a separate conf file for their akka stanza which was loaded in separately. That keeps the config-ness to it all but does put it in a slightly less accessible location.

Although now that I say this i think my dispatcher point isn't true, I don't think you'd be able to remove one via an override. It's like the issue we had w/ the local backend.